examples/flutter_gallery/lib/demo/cards_demo.dart

```diff
@@ -74,7 +74,12 @@ class TravelDestinationItem extends StatelessWidget {
                   new Positioned(
                     bottom: 16.0,
                     left: 16.0,
-                    child: new Text(destination.title, style: titleStyle)
+                    child: new DefaultTextStyle(
+                      softWrap: false,
+                      overflow: TextOverflow.ellipsis,
+                      style: titleStyle,
+                      child: new Text(destination.title)
```

Comment:
No need to use a default text style here.  You can just give these parameters to the `Text` directly.